src/autocomplete/databases/redis/tests/get/get.test.ts

```diff
@@ -0,0 +1,19 @@
+import {parseRedisQueryWithoutCursor} from '../../index';
+
+test('should not report errors on GET command', () => {
+    const autocompleteResult = parseRedisQueryWithoutCursor('GET test');
+
+    expect(autocompleteResult.errors).toHaveLength(0);
+});
+
+test('should not report errors on multiple GET commands', () => {
+    const autocompleteResult = parseRedisQueryWithoutCursor('GET test\nGET 123');
+
+    expect(autocompleteResult.errors).toHaveLength(0);
+});
```

Comment:
I think it's better to have a separete multiple commands test. Because in the end we want to test `commands` rule https://github.com/gravity-ui/websql-autocomplete/pull/207/commits/3fd0a0952a60ffd894d03f6ee326ae50bade0c03#diff-b96534631ee82ffc85910f193fcacb74b496e3324eef92e1d7121103f2321683R16